<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>501</width>
    <height>255</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>501</width>
    <height>255</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>501</width>
    <height>255</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>TaskTicker</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>501</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>48</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>TaskTicker 0.1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btnEngage">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>120</y>
      <width>94</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Engage</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnNewTask">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>120</y>
      <width>94</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>New Task</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnViewTasks">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>120</y>
      <width>94</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>View Tasks</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnExit">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>120</y>
      <width>94</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>501</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuControls">
    <property name="title">
     <string>Actions</string>
    </property>
    <addaction name="act_engage"/>
    <addaction name="act_disengage"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionTasks"/>
    <addaction name="actionTags"/>
    <addaction name="actionPreferences"/>
   </widget>
   <addaction name="menuControls"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="act_engage">
   <property name="text">
    <string>Engage</string>
   </property>
  </action>
  <action name="act_disengage">
   <property name="text">
    <string>Disengage</string>
   </property>
  </action>
  <action name="actionTags">
   <property name="text">
    <string>Tags</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
  <action name="actionTasks">
   <property name="text">
    <string>Tasks</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
